exec/java-exec/src/test/java/org/apache/drill/exec/compile/TestLargeFileCompilation.java

```diff
@@ -154,4 +158,20 @@ public void testProject() throws Exception {
     testNoResult(ITERATION_COUNT, LARGE_QUERY_SELECT_LIST);
   }
 
+  @Test
+  public void testSelectAllFromFileWithManyColumns() throws Exception {
+    File path = new File(BaseTestQuery.getTempDir("json/input"));
```

Comment:
You are right, with this changes both tests testEXTERNAL_SORT and testTOP_N_SORT work. So I enabled them.